javascriptv3/example_code/bedrock-runtime/models/ai21_labs_jurassic2/jurassic2.js

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fileURLToPath } from "url";
+
+import { FoundationModels } from "../../config/foundation_models.js";
+import { defaultProvider } from "@aws-sdk/credential-provider-node";
+import {
+  BedrockRuntimeClient,
+  InvokeModelCommand,
+} from "@aws-sdk/client-bedrock-runtime";
+
+/**
+ * @typedef {Object} Data
+ * @property {string} text
+ *
+ * @typedef {Object} Completion
+ * @property {Data} data
+ *
+ * @typedef {Object} ResponseBody
+ * @property {Completion[]} completions
+ */
+
+/**
+ * Invokes an AI21 Labs Jurassic-2 model.
+ *
+ * @param {string} prompt - The input text prompt for the model to complete.
+ * @param {string} [modelId] - The ID of the model to use. Defaults to "ai21.j2-mid-v1".
+ * @returns {Promise<string[]>} The inference response from the model.
```

Comment:
In most cases, I would recommend leaving off `@returns`. When you declare it explicitly like this, it overrides whatever might have been inferred by the interpreter. I've had this lead to mismatches between what's declared here and what's actually being returned. Nasty bug hunt.

Reply:
I'll simply remove them